__tests__/server/user-routes.spec.js

```diff
@@ -18,80 +24,117 @@ describe('Checks user answered routes', () => {
     // check if each of the recordId is one existing questionId
     answeredQuestions.forEach(question => expect(questionIDs.includes(question.ID)).toBe(true))
   })
-  it('should return the empty array of the answered questions of user2', async () => {
-    const res = await request.get(`/api/user/${user2ID}/answered`)
+  it('should return the empty array of the answered questions of user test2', async () => {
+    await testSession.post('/api/auth/login').send({ Email: 'a2@b.com', Password: 'password'})
+    const res = await testSession.get('/api/user/answered')
     expect(res.status).toBe(200)
     expect(res.body.length).toBe(0)
   })
-  it('should return updated ids of the answered questions when multiple questions are added sequentially', async () => {
-    let res
-    res = await request.get(`/api/user/${user2ID}/answered`)
+  afterAll(async () => {
+    clearFieldsInSingleRecord(userBaseName, user2Id, answeredFieldName)
+  })
+}) 
+
+describe('Checks to see if user one answer updates correctly', () => {
+  const answeredFieldName = 'Answered'
+  let questions
+  let questionIDs
+  // get the question ids before the tests
+  beforeAll(async () => {
+    const questionsRes = await request.get('/api/questions')
+    expect(questionsRes.status).toBe(200)
+    questions = questionsRes.body
+    questionIDs = questions.map(question => question.ID)
+    
+  })
+  // the following test might modify `answered` field of user test2
+  beforeEach(async () => {
+    clearFieldsInSingleRecord(userBaseName, user2Id, answeredFieldName)
+    await testSession.post('/api/auth/login').send({ Email: 'a2@b.com', Password: 'password'})
+    const res = await testSession.post('/api/user/answer').send({ questionId: questionIDs[7]})
+  })
+  it('should return updated user w/ new question in answered field', async () => {
+    res = await testSession.get('/api/user/answered')
     expect(res.status).toBe(200)
-    expect(res.body.length).toBe(0)
-    const numOfQuestionsAdded = Math.min(4, questions.length)
+    expect(res.body.length).toBe(1)
+    console.log(res.body)
+    const answered = res.body//.Answered
+    expect(answered.includes(questions[7]))
+  }, 3000 ) // needs a little time for the change go through the DB
+  afterAll(async () => {
+    clearFieldsInSingleRecord(userBaseName, user2Id, answeredFieldName)
+  })
+})
+
+describe('Checks to see if user multiple answers update correctly', () => {
+  const answeredFieldName = 'Answered'
+  let questions
+  let questionIDs
+  let numOfQuestionsAdded
+  // get the question ids before the tests
+  beforeAll(async () => {
+    const questionsRes = await request.get('/api/questions')
+    expect(questionsRes.status).toBe(200)
+    questions = questionsRes.body
+    questionIDs = questions.map(question => question.ID)
+  })
+  // the following test might modify `answered` field of user test2
+  beforeEach(async () => {
+    clearFieldsInSingleRecord(userBaseName, user2Id, answeredFieldName)

```

Comment:
this should be moved back to the afterAll and changed to user test not user test2